tests/Serializer/JsonSerializationTest.php

```diff
@@ -203,34 +205,18 @@ public function getPrimitiveTypes()
                 'type' => 'boolean',
                 'data' => true,
             ),
-            array(
```

Comment:
All these are removed, type coertion of this sort is incompatible with strict types by design.